commands/voice.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2015 The tgbot Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package commands
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/url"
+	"os"
+	"regexp"
+)
+
+func SetResourceUrl(lang, text string) string {
```

Comment:
It should be called setResourceUrl. If it starts with a capital letter, the function would be exported and it is an internal helper.

Put the function at the end of the file, after Run.